pkg/sdk/config.go

```diff
@@ -214,7 +214,6 @@ func GetConfigFileName() (string, error) {
 
 // TODO(SNOW-1787920): improve TOML parsing
 type ConfigDTO struct {
-	Account                        *string             `toml:"account"`
```

Comment:
nit (question): what is our current policy in toml parsing: ignore unknown or fail on unknown? If it's the latter I would add a sentence in migration guide which reminds that these fields should be removed both from config but also from the toml config file.

Reply:
We ignore fields that are not in ConfigDTO. We fail when there is a data type mismatch on a known field. Added more unit tests for that. But still, I added more examples in the migration guide.